docsite/rst/intro_inventory.rst

```diff
@@ -203,16 +204,38 @@ mentioned::
       The default ssh user name to use.
     ansible_ssh_pass
       The ssh password to use (this is insecure, we strongly recommend using --ask-pass or SSH keys)
+    ansible_connection
+      Connection type of the host. Candidates are local, ssh or paramiko.  The default is paramiko before Ansible 1.2, and 'smart' afterwards which detects whether usage of 'ssh' would be feasible based on whether ControlPersist is supported.
+    ansible_ssh_private_key_file
+      Private key file used by ssh.  Useful if using multiple keys and you don't want to use SSH agent.
+
+Privilege escalation (see :doc:`Ansible Privilege Escalation<become>` for further details). Since Ansible 1.9 you should use *ansible_become*::
+
```

Comment:
If we remove ansible_su/do from there I don't see anywhere else where they are documented.